.github/workflows/gitlab-runners-ubi8-git-publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: ubi8-git-publish
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - gitlab-runners/ubi8-git/version.json
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+      quay_org: redhat-cop
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get image tags
+        id: image_tags
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          VERSION=$(jq -r '.version' ${context}/version.json)
+          if [ "${VERSION}" == "latest"]; then
+            echo latest
+          else
+            echo "latest,${VERSION}"
+          fi
+      - name: Build and publish image to Quay
+        uses: docker/build-push-action@v1
+        with:
+          path: ${{ env.context }}
+          registry: quay.io
+          repository: ${{ env.quay_org }}/${{ env.image_name }}
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
```

Comment:
`username: ${{ secrets.REGISTRY_PASSWORD }}`

no need to make it quay specific.